wiki/Ranking_Criteria/zh.md

```diff
@@ -1,300 +1,303 @@
-- **尽量避免任何明显的性能问题。即使优化过，保持帧速率一致对于打图体验也是至关重要的。** 请在摸图时测试谱面来确认这一点。
-- **避免使用故事板音效，在打图过程中容易与敲击音效混淆。** 这违背了听觉反馈这一概念，因为故事板音效将独立于玩家的操作而自动播放。
-- **避免使用不合逻辑、相互冲突和已经作废的命令。** 游戏无法正常执行间隔重叠、结束时间早于开始时间或无法成功触发的这类命令。应删除或修正。
-- **如果谱面中包含宽屏故事板，则应启用 `Widescreen support`（宽屏支持）。** 反之，如果故事板的大小设计为 4:3 分辨率，则应关闭此功能。如果没有故事板，此设置不会影响谱面中的任何内容。
-- **请尽可能优化故事板，** 无论如何都要做到这一点。
-  - **避免精灵图或谱面背景在渲染时被完全遮挡至不可见。** 出于性能考虑，当元素不可见时，最好将它们隐藏。如要隐藏谱面的背景，请将相同的背景图像作为精灵图，`Background` 或 `0` 作为第二个参数，然后隐藏掉它。
-  - **避免精灵图在整个渲染过程中离开屏幕或者阻碍视线。** 在这种情况下，应该裁剪图像的不可见部分，除非它们对故事板中的效果来说是必要的。
-  - **避免故事板图像周围不必要的透明。** 为了提高性能，应在保留所需效果下尽可能多地裁剪图像。
-  - **若要多次使用某条命令，请使用循环指令，除非这样做会影响视觉效果。** 使用 Loop 指令通常会大量减少代码行数，从而减少文件大小。
-  - **当使用一个通用命令可以实现相同的效果时，请避免使用两个坐标轴分量命令。** 使用单条命令而不是两条命令，能减少文件大小。
-  - **使用最节省空间的图像文件格式，同时保持合理的质量。** 不同于 `.jpg` 格式，`.png` 格式由于无损压缩方式，文件通常更大。
-  - **避免任何重复的图像文件。** 完全相同内容的两个图像会增加不必要的文件大小。
-  - **避免同时激活多个不可见的精灵图。** 无论元素是否可见，游戏仍然会处理元素的命令。如果这种元素不可见情况的持续时间较长，请创建一个新的元素在其重新出现时启用。
-  - **在同一个精灵图上使用相同类型的多个指令时，请使它们的激活时间间隔至少保留 16 毫秒。** 每秒 60 个命令足以使精灵图视觉上平滑过渡，且可以减小文件大小，减少加载时间。
-  - **由触发器激活的精灵图要在使用后隐藏。** 触发器会从第一个激活命令启用，保持激活状态直到谱面结束，所以最好在不使用后隐藏它。
+- **故事板图片所包含的总像素数量不能超过 1700 万，这是为了保证大部分电脑能在短时间内加载完成。** 此外，你需要根据故事板编辑器内部最大尺寸（854 x 480 osu! 像素）来缩放图片。
+- **谱面在加载时，不能引发解析错误。** 即解析器无法读取故事板指令的某一部分。
+- **谱面的`宽屏支持` (Widescreen Support) 设置必须保持一致。** 但如果不同难度的故事板是为不同宽高比设计，则不受这一规定的限制。
+
+### 指南
+
+- **在歌曲结束后，不应存有激活精灵图和命令。** 为了某些故事板的特定效果需要，精灵图的渲染时长可以延长到歌曲结束后一段时间，但不应过长。
+- **在含有旋转指令的故事板图片周围绘制 1 像素宽的透明边框。** osu! 没有图片的抗锯齿算法。因此，如果用作旋转的故事板图片边缘非常锐利，旋转中的锯齿将会非常明显。
+- **尽量保持故事板播放时帧率稳定，尽量避免任何明显的性能问题。** 请在摸图时测试谱面来确认这一点。
+- **避免使用在游玩过程中容易与打击音效混淆的故事板音效。** 这违背了听觉反馈这一概念，因为故事板音效将独立于玩家的操作而自动播放。
+- **避免使用不合逻辑、相互冲突或已经作废的命令。** 游戏无法正常执行激活时间重叠、结束时间早于开始时间等无法成功触发的命令。应删除或修正它们。
+- **如果需要制作适配宽屏的故事板，则应启用`宽屏支持` (Widescreen Support) 。** 如果故事板设计为 4:3 分辨率，则应关闭此功能。如果没有制作故事板，此设置不会影响谱面中的任何内容。
+- **尽量优化故事板。** 无论如何都要做到这一点。
+  - **避免精灵图或谱面背景图片在渲染时，被遮挡至完全不可见。** 出于性能考虑，当元素不可见时，最好将它们淡化为透明。如果要将谱面的背景图片淡化为透明，请将相同的背景图片名称作为精灵图，`Background` 或 `0` 作为第二个参数，然后使用淡化命令，隐藏掉它。
+  - **避免精灵图在整个渲染过程中离开屏幕或者阻碍视线。** 如果故事板图片的屏幕外不可见部分自始至终都没有展示在屏幕中，就应该裁剪掉这一部分。
+  - **避免在故事板图片周围留有大量的透明像素。** 为了提高性能，应在保证图片主体完整展示的前提下，尽可能多地裁剪图片。
+  - **若要多次使用某条命令，请使用循环 (Loop) 指令，除非这样做会影响视觉效果。** 使用循环指令能大量减少代码行数，从而缩减故事板文件大小。
+  - **当使用一个常用命令可以实现相同的效果时，请不要混合使用两个坐标轴分量命令。** 使用单条命令而不是两条命令，能缩减故事板文件大小。
+  - **使用最节省空间的图片文件格式，同时保持合理的质量。** 不同于 `.jpg` 格式，`.png` 格式由于采用了无损压缩方式，占用的存储空间通常更大。
+  - **避免文件夹内的图片重复。** 完全没有必要将内容完全相同的两张图片放在一起使用。
+  - **避免同时激活多个不可见的精灵图。** 无论元素是否可见，游戏仍然会计算并处理命令。如果这个精灵图长时间不可见，请关闭此精灵图，并在可见时间处创建一个新的精灵图。
+  - **在同一个精灵图上使用多个同类型指令时，指令开启时间间隔至少保留 16 毫秒。** 每秒 60 个命令足以使精灵图在视觉上平滑过渡。这也可以缩减故事板文件大小，减少故事板加载时间。
+  - **由触发器 (Triggers) 激活的精灵图要在使用完毕后淡化为透明。** 触发器会从第一个激活命令开启，保持激活状态直到谱面结束。所以最好在不使用后隐藏这个精灵图。
+
+## 备注
+
+[^角色配音演员与配音演员]:当某虚拟角色的配音演员以该角色的身份演唱歌曲时，应使用 `CV` 标注该角色配音演员；如果演唱歌曲的艺术家并不是此虚拟角色的配音演员，则应使用 `VO` 标注该配音演员。
+[^最大尺寸]:如果该背景图片也用作故事板的一部分，则应遵循[规定的故事板图片最大尺寸](/wiki/Ranking_Criteria#规定.7)。
+[^普通打击音效与附加打击音效]: 使用[谱面音效](/wiki/Beatmapping/Hitsound)时，普通打击音效 (*hitnormal*) 总会播放，并可以与口哨、镲、拍手这类附加打击音效 (*additions*) 组合使用。
```

Comment:
it's better to not translate these identifiers because they're not visible anyway

```suggestion
[^character-voice-actor]: 当某虚拟角色的配音演员以该角色的身份演唱歌曲时，应使用 `CV` 标注该角色配音演员；如果演唱歌曲的艺术家并不是此虚拟角色的配音演员，则应使用 `VO` 标注该配音演员。
[^maximum-dimensions]: 如果该背景图片也用作故事板的一部分，则应遵循[规定的故事板图片最大尺寸](/wiki/Ranking_Criteria#规定.7)。
[^normal-vs-addition]: 使用[谱面音效](/wiki/Beatmapping/Hitsound)时，普通打击音效 (*hitnormal*) 总会播放，并可以与口哨、镲、拍手这类附加打击音效 (*additions*) 组合使用。
```